tree/ntuple/v7/inc/ROOT/RNTuple.hxx

```diff
@@ -182,22 +181,22 @@ public:
                                               std::string_view storage,
                                               const RNTupleReadOptions &options = RNTupleReadOptions());
    static std::unique_ptr<RNTupleReader>
-   Open(RNTuple *ntuple, const RNTupleReadOptions &options = RNTupleReadOptions());
+   Open(RNTuple *ntuple, const RNTupleReadOptions &options =
+                            RNTupleReadOptions()); /// The caller imposes a model, which must be compatible with the
+                                                   /// model found in the data on storage.
```

Comment:
I guess `clang-format` butchered up the comment here? Maybe put an empty line between...